server/tag/tag.controller.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { Tag, Repository, User } = require('../shared/database');
+
+function list({ user }, res) {
+  const include = [{
+    model: Repository,
+    as: 'repositories',
+    attributes: ['id'],
+    required: true,
+    include: [{ model: User, attributes: ['id'], where: { id: user.id } }]
+  }];
+  const options = {};
+  if (!user.isAdmin()) options.include = include;
```

Comment:
Rather use ternary and don't define include if its not needed. (`const options = user.isAdmin ? {} : ......`)